core/frontend/src/tile/map/MapTilingScheme.ts

```diff
@@ -188,8 +188,10 @@ export abstract class MapTilingScheme {
   }
 
   /** @alpha */
-  private ecefToPixelFraction(point: Point3d, applyTerrain: boolean): Point3d {
+  private ecefToPixelFraction(point: Point3d, applyTerrain: boolean): Point3d | undefined {
     const cartoGraphic = Cartographic.fromEcef(point)!;
```

Comment:
```suggestion
    const cartoGraphic = Cartographic.fromEcef(point);
```
Can get rid of non-null assertion now that you're handling the null case.